src/gval/comparison/compute_continuous_metrics.py

```diff
@@ -17,7 +17,7 @@
 
 from gval import ContStats
 from gval.utils.schemas import Metrics_df, Subsample_identifiers, Sample_identifiers
-from gval.utils.loading_datasets import _check_dask_array, _convert_to_dataset
+from gval.utils.loading_datasets import _convert_to_dataset
 
 
 def _get_selected_datasets(
```

Comment:
I know this function existed but based on my understanding would something like `_get_data_mask` be a more appropriate name for it? The same could be said about the top level description in the docstring.

Reply:
I went ahead and changed the name of the function and docstring.